Makefile

```diff
@@ -120,51 +142,68 @@ DEPENDS	:=	$(OFILES:.o=.d)
 #---------------------------------------------------------------------------------
 # main targets
 #---------------------------------------------------------------------------------
-all: $(OUTPUT).gcb $(OUTPUT).vgc $(OUTPUT).gci $(OUTPUT)_xz.gci
 
+#---------------------------------------------------------------------------------
+# Base DOL (PicoBoot)
+#---------------------------------------------------------------------------------
 $(OUTPUT).dol: $(OUTPUT).elf
 $(OUTPUT).elf: $(OFILES)
 
 $(OFILES_SOURCES) : $(HFILES)
 
-%.gcb: %.dol
-	@echo pack IPL ... $(notdir $@)
-	@cd $(PWD); ./dol2ipl.py ipl.rom $< $@
-
+#---------------------------------------------------------------------------------
+# GCI
+#---------------------------------------------------------------------------------
 %.gci: %.dol
-	@echo dol2gci ... $(notdir $@)
+	@echo converting DOL to GCI ... $(notdir $@)
```

Comment:
Were the existing messages not explicit enough? I don't mind the change, just curious.
Either way, instead of progressive tenses I think I'd rather stick to imperative form: "converting" => "convert". Thoughts?

Reply:
There were a few instances in which the same message was echoed for different tasks. I made them a bit more verbose for clarity and uniqueness. Don't have a preference on tense. There was a mix before ("pack IPL ..." and "linking ...") and I went with progressive to stay consistent with devkitPro (which I now realize is inconsistent ha "output ..." and "linking ...").

Reply:
In that case you could make it even more explicit, something like "generate memory card save file". It's really no big deal, as long as all the messages are unique enough to debug the build script, we're good. We're not designing a user-facing interface here.

Reply:
Exactly. I just wanted them to be unique so I could differentiate which target was executed while debugging. I'll change to imperative.